docs/source/CustomizingTheWorkflow/DASystem.rst

```diff
@@ -930,53 +927,67 @@ The ``submit_cycle.sh`` script first exports the required paths and loads the re
 
    *Flowchart of 'submit_cycle.sh'*
 
-As the script loops through the steps in the process for each cycle, it reads in the DA settings and selects a run type --- either DA or ``openloop`` (which skips DA). Required DA settings include: DA algorithm choice, directory paths for JEDI, Land_DA (where the ``do_landDA.sh`` script is located), JEDI's input observation options, DA window length, options for constructing ``yaml`` files, etc. 
+As the script loops through the steps in the process for each cycle, it reads in the DA settings and selects a run type --- either DA or ``openloop`` (which skips DA). Required DA settings include: DA algorithm choice, directory paths for JEDI, Land_DA (where the ``do_landDA.sh`` script is located), JEDI's input observation options, DA window length, options for constructing ``.yaml`` files, etc. 
 
 Next, the system designates work and output directories and copies restart files into the working directory. If the DA option is selected, the script calls the ``vector2tile`` function and tries to convert the format of the Noah-MP model in vector space to the JEDI tile format used in :term:`FV3` cubed-sphere space. After the ``vector2tile`` is done, the script calls the data assimilation job script (``do_landDA.sh``) and runs this script. Then, the ``tile2vector`` function is called and converts the JEDI output tiles back to vector format. The converted vector outputs are saved, and the forecast model is run. Then, the script checks the existing model outputs. Finally, if the current date is less than the end date, this same procedure will be processed for the next cycle.
 
 .. note:: 
 
-   The v1.0.0 release of Land DA does not support ensemble runs. Thus, the first ensemble member (``mem000``) is the only ensemble member. 
+   The |latestr| release of Land DA does not support ensemble runs. Thus, the first ensemble member (``mem000``) is the only ensemble member. 
 
-Here is an example of configuration settings file, ``settings_cycle``, for the ``submit_cycle`` script:
+Here is an example of the configuration settings file, ``settings_DA_cycle_*``, for the ``submit_cycle.sh`` script:
 
 .. code-block:: console
    
-   export exp_name=DA_IMS_test
-   STARTDATE=2016010118
-   ENDDATE=2016010318
+   # experiment name
+   export exp_name=DA_ERA5_test
+   #export BASELINE=hera.internal
```

Comment:
@gspetro-NOAA please delete this line

Reply:
line 944

Reply:
I just kept the previous example because it it still relevant as an example, but I'll go ahead and update.